.github/workflows/linters.yml

```diff
@@ -0,0 +1,48 @@
+name: Linters
+
+on: pull_request
+
+env:
+  FORCE_COLOR: 1
```

Comment:
- [ ] Please Branch should have a name of a feature or action (e.g. `created_sidebar` or `added_comments_section`).
![image](https://user-images.githubusercontent.com/74744787/164072441-a7235ad0-43bd-478f-8e29-87e31d3eb86c.png)